.changesets/add-error-helpers.md

```diff
@@ -0,0 +1,17 @@
+---
+bump: "patch"
+type: "add"
+---
+
+Add error helpers. These helpers simplify reporting errors to AppSignal that are not handled automatically by your framework, alongside the current instrumented context or separately. For example, to report an error from an `except` clause as a separate sample:
+
+```python
+from appsignal import send_error
+
+def index():
+  cookies = put_hand_in_cookie_jar()
+  except HandStuckError as error:
+    send_error(error)
+```
+
+More information about error helpers is available in our documentation.
```

Comment:
```suggestion
More information about [error helpers](https://docs.appsignal.com/python/custom-instrumentation/exception-handling.html) is available in our documentation.
```

I suggest we link to this location. It doesn't exist yet, but we have a similar page for other integrations, so we should make one for Python too.